### DISCUSSION TITLE
Search in inbox by title

### DISCUSSION THREAD

**Discussion by PackmanDude** [community/community#20461] [Other Feature Feedback, Questions, & Ideas]
How can I search like in the global search feature, by name (title) of the notification in Inbox, Done, et cetera? There are only a few filters and not a simple one, which would make it more user-friendly.

**Comment by JohnnyArachnid** [comment]
To enhance the search functionality in your notification system, consider implementing a dedicated search bar for real-time filtering as users type, and expand filter options to include title, date range, and notification type. Introduce advanced search options with logical operators, provide autocomplete suggestions based on past notifications, and highlight matched keywords in results for easier scanning. Incorporate pagination or infinite scroll for navigation, allow sorting of results by criteria like date and relevance, and ensure mobile responsiveness for a seamless experience on smaller screens. These improvements will create a more user-friendly interface for finding notifications.

**Reply by PackmanDude** [reply]
I'm talking about GitHub UI.

**Reply by JohnnyArachnid** [reply]
In GitHub UI there's no direct way in my knowledge to search by the title of a notification, you can filter by type (like "Mention" or "Pull Request") using the available filters. Entering specific keywords related to your notifications in the search bar can also help, such as repository names or issue titles. Additionally, combining filters can narrow down your search results, and sorting by date may assist in locating notifications more quickly. If you're struggling to find a specific notification, using your browser's search function (Ctrl + F or Command + F) on the Notifications page can be a useful workaround. Although these methods don't fully replicate a global search feature, they are fine enough.
